std/portable/index.d.ts

```diff
@@ -230,11 +230,15 @@ declare namespace f32 {
   /** Largest representable value. */
   export const MAX_VALUE: f32;
   /** Smallest normalized positive value. */
-  export const MIN_POSITIVE_VALUE: f32;
+  export const MIN_NORMAL_VALUE: f32;
   /** Smallest safely representable integer value. */
   export const MIN_SAFE_INTEGER: f32;
   /** Largest safely representable integer value. */
   export const MAX_SAFE_INTEGER: f32;
+  /** Positive infinity value. */
+  export const POSITIVE_INFINITY: f32;
+  /** Negative infinity value. */
+  export const NEGATIVE_INFINITY: f32;
```

Comment:
So, previously these were present on `F32` analogous to JS's `Number` (but not on `f32`), with the other members of `F32` being defined as

```ts
class F32 {
  ...
  static readonly MAX_SAFE_INTEGER: f32 = f32.MAX_SAFE_INTEGER;
  ...
}
```

so perhaps it makes sense to also update

```ts
class F32 {
  ...
  static readonly POSITIVE_INFINITY: f32 = f32.POSITIVE_INFINITY;
  ...
  static readonly NEGATIVE_INFINITY: f32 = f32.NEGATIVE_INFINITY;
  ...
}
```

for consistency?

Reply:
Yeah. Also added f32.NaN and f64.NaN for consistency